### PR TITLE
Fixed UI in the Epic quest and Books section

### PR DESCRIPTION
**Pull request by ashvin89**
|PR checklist|Status|
|-----|:------:|
|Is issue created in GitHub?|[Yes](#88 )|Yes|
|Is code analysis task executed and issues resolved in the Java/Kotlin code?|NA|
|Is code analysis task executed and issues resolved in the CPP/native code?|No|
|Is the version code set in the AndroidManifest.xml or build.gradle file to 1 more than the version which has already been published?|NA|
|Is code commenting done properly for the changes made?|Yes|
|Is correct variable naming convention followed?|Yes|
|Is auto formatting shortcut Ctrl+Alt+Shift+L executed in all files?|Yes|
|Are the message strings used in the application taken from strings.xml file?|NA|
|Are the permissions which are not required removed from AndroidManifest.java file?|NA|
|Is the _uses-feature_ tag used for only must have feature in AndroidManifest.java file?|NA|
|Are the generated files correctly configured in .gitignore?|NA|
|Are the release signing configurations removed from the gradle files?|NA|
|Is commented code removed from all files?|Yes|
|Are updated images compressed (< 100 KB)?|NA|
|Ensured there are no typos in any string. For example: Variable/method names, hardcoded strings, commit messages, PR titles, issues, code comments?|Yes|

### CODE REVIEW

File: EnglishWebRoot/school/script/load.js [hunk 1]
```diff
@@ -1285,6 +1285,32 @@ function xmlLoaded(curXML) {
         // Should give enough time for the viewport meta to finally parse in the browser.
         $("body").css("display", "block");
         loadAssets();
+        // Fitting images to all the device according to their screen width and screen height 
+
```
Comment (Shailin) [inline EnglishWebRoot/school/script/load.js]: Remove unnecessary spaces if not required

File: EnglishWebRoot/school/pubbly_engine/latest/js/run/dom.js [hunk 2]
```diff
@@ -155,47 +155,37 @@ function PubblyDom(xml, environment) {
         $("#main").addClass("transformCenterCont");
         $("#main > div").addClass("transformCenter");
 
-        function scalingYCoordinate(screenAttributes) {
-            screenAttributes.style.height = '100%';
-            //  Converting 100% to pixel
-            var totalpixel = parseFloat(window.getComputedStyle(screenAttributes).height);
-            screenAttributes.style.height = domHeight;
+        function scalingYCoordinate() {
+            var totalpixel = info.height * viewportScaleNoMargins;
```
Comment (Shailin) [inline EnglishWebRoot/school/pubbly_engine/latest/js/run/dom.js:160]: Add code comments

File: EnglishWebRoot/school/script/load.js (hunk 1 above)
Comment (ashvin89) [reply EnglishWebRoot/school/script/load.js]: Removed spaces

File: EnglishWebRoot/school/pubbly_engine/latest/js/run/dom.js (hunk 2 above)
Comment (ashvin89) [reply EnglishWebRoot/school/pubbly_engine/latest/js/run/dom.js:160]: Added code comments

**Review by Shailin — APPROVED** [review]

**Review by mathewnik90 — APPROVED** [review]

**Pull request merged**